packages/xsnap/src/avaAssertXS.js

```diff
@@ -168,17 +168,29 @@ function createHarness(send) {
 /**
  * @param {*} exc
  * @param {Expectation} expectation
- * @returns {boolean}
+ * @returns {null | { expected: unknown, actual: unknown }}
```

Comment:
@erights has advised pretending `null` doesn’t exist and favoring `| void` generally.

Reply:
pointer to said advice? I'd like to add it to https://github.com/Agoric/agoric-sdk/wiki/Code-Review-Checklist , along with rationale.

I find it odd that we'd prefer `undefined` over `null`, since JSON groks `null`.